src/partner/ducks/currentEvent.ts

```diff
@@ -0,0 +1,125 @@
+import { IEvent, event } from "src/partner/models/Event";
+
+export interface IState {
+  events: IEvent[];
+  isLoading: boolean;
+  error: any;
+  currentEvent: IEvent;
+  cancelEvent: boolean;
+  openModal: boolean;
+  editEvent: boolean;
+}
+
+export interface IActions {
+  type: String;
+  payload?: any;
+  error?: boolean;
+  meta?: any;
+}
+
+export const initialState: IState = {
+  events: [],
+  isLoading: false,
+  error: undefined,
+  currentEvent: event(),
+  cancelEvent: false,
+  openModal: false,
+  editEvent: false,
+};
+
+const CANCEL_EVENT = "partner/views/currentEvent/CANCEL_EVENT";
+const CLOSE_MODAL = "partner/views/currentEvent/CLOSE_EVENT";
+const CLOSE_MODAL_CANCEL_EVENT = "partner/views/currentEvent/CLOSE_MODAL_CANCEL_EVENT";
+const CREATE_EVENT = "partner/views/currentEvent/CREATE_EVENT";
+const UPDATE_EVENT = "partner/views/currentEvent/UPDATE_EVENT";
+const SHOW_MODAL = "partner/views/currentEvent/SHOW_MODAL";
+const SHOW_EDIT_MODAL = "partner/views/currentEvent/SHOW_EDIT_MODAL";
+const SHOW_MODAL_CANCEL_EVENT = "partner/views/currentEvent/SHOW_MODAL_CANCEL_EVENT";
+const START_FETCHING = "partner/views/currentEvent/START_FETCHING";
```

Comment:
rename to `FETCH_CURRENT_EVENT_START`, `FETCH_CURRENT_EVENT_SUCCESS`, `FETCH_CURRENT_EVENT_FAIL`